File: TareaSemana4/Paciente.cs
using System;

namespace AgendaClinicaApp
{
    // Estructura para guardar información del paciente
    public struct Paciente
    {
        public string Cedula { get; set; }
        public string Nombre { get; set; }
        public string Motivo { get; set; }
        public string FechaTurno { get; set; }
        public string HoraTurno { get; set; }
    }
}

File: TareaSemana4/Paciente.java
public class Paciente {
    private String cedula;
    private String nombre;
    private String motivo;
    private String fecha;
    private String hora;

    public Paciente(String cedula, String nombre, String motivo, String fecha, String hora) {
        this.cedula = cedula;
        this.nombre = nombre;
        this.motivo = motivo;
        this.fecha = fecha;
        this.hora = hora;
    }

    public String getCedula() {
        return cedula;
    }

    public void mostrarDatos() {
        System.out.println("Cédula: " + cedula + ", Nombre: " + nombre +
                           ", Motivo: " + motivo + ", Fecha: " + fecha + ", Hora: " + hora);
    }
}
